components/cdc/src/endpoint.rs

```diff
@@ -653,27 +673,37 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
 
             // TODO: send a message to raftstore would consume too much cpu time,
             // try to handle it outside raftstore.
-            for (region_id, observe_id) in regions {
+            for (region_id, observe_id, remote_lease) in regions {
                 let scheduler_clone = scheduler.clone();
+                let schedule_min_ts =
+                    move |remote_lease| match scheduler_clone.schedule(Task::MinTS {
```

Comment:
To avoid confusion, please use another name.
```suggestion
                    move |lease| match scheduler_clone.schedule(Task::MinTS {
```